_posts/2022-01-05-green-lab-experiment.md

```diff
@@ -0,0 +1,145 @@
+
+There are some noises and outliers in our experimental data. For the extreme noise such as non-positive packets' number or size and non-positive energy consumption, we cut off these data for data cleansing. These data are collected incorrectly in the procedures, probably in AndroidRunner or WireShark. And for some outliers like low consumption and low network traffics in a 1080p quality setting, we keep this for these two reasons. First, the data can be correct since some video playback can be short and do not have any advertisements. Second, the data plays a crucial role in statistics analysis for rejecting the null hypothesis. We can not remove them because they might suggest no impact on the consumption or network traffic under different quality settings.
+
+### Hypothesis Testing
+
+From the previous analysis, the distribution of power consumption and network packages are not from the normal distribution. The distributions are similar in both Block One(YouTube) and Block Two(Twitch). We apply the non-parametric test based on our experiment design for both Blocks. For the test selection, since the dependent variables(energy consumption and network traffic) are tested in different situations with different levels of qualities, we use the Kruskal-Wallis non-parametric tests since they fit most in this test. 
+
+1. YouTube
+
+YouTube provide four quality setting options: 360p, 480p, 720p 1080p. We set energy consumption and network traffic data with these different quality options as input in the test. By setting 1080p as value 3 and 360p as 1, we got the p-value of energy consumption and for network packets are both less than 2.2e-16, which reject our null hypothesis, indicating that energy consumption and network packets are different under different quality settings in block one.
+
+2. Twitch
+
+We use the four quality setting options that Twitch provides: 360p, 480p, 720p 1080p. Similar to Block One with YouTube, the experimental structure and the levels of quality are the same. So we apply the Kruskal-Wallis non-parametric tests on the energy consumption and network packages for Twitch. Using the same numeric transformation as block one. By setting 1080p as value 3 and 360p as 1, we got a p-value of 6.154e-16 for energy consumption and 7.219e-14 for network packets, so that null hypothesis is rejected, indicating that energy consumption and network packets are different under different quality settings in block two.
+
+### Effect Size Estimation
+
+We investigate the effect size by using Cliff’s delta measure. We found an effect size of -0.83 and -0.99, for all the different video qualities in the energy consumption and network from YouTube and Twitch. According to Vargha and Delaney (2000) \cite{sagepub} the value of the effect size can be interpreted as small, >= 0.11, medium, >= 0.28, large, >= 0.43. Since our effect size is -0.83 and -0.99, this means we have a large effect size, because the values are in inverse.
```

Comment:
replace all citations using the \cite{} Latex code with a direct link to the cited paper